spec/Gaufrette/Adapter/FlysystemSpec.php

```diff
@@ -31,11 +31,46 @@ function it_reads_file(AdapterInterface $adapter)
         $this->read('filename')->shouldReturn('Hello.');
     }
 
+    function it_throws_file_not_found_exception_when_trying_to_read_an_unexisting_file(AdapterInterface $adapter)
+    {
+        $adapter->read('filename')->willThrow(new FileNotFoundException('filename'));
+
+        $this->shouldThrow('Gaufrette\Exception\FileNotFound')->duringread('filename');
```

Comment:
Shame on me :hammer: removing the `composer.lock` file and running `composer install` again did the trick :) Thank you :+1: